lib/money-rails/active_record/monetizable.rb

```diff
@@ -89,30 +89,16 @@ def monetized_attributes
             #   monetize :price_in_a_range_cents, :disable_validation => true
             if validation_enabled = MoneyRails.include_validations && !options[:disable_validation]
 
-              subunit_validation_options =
-              unless options.has_key? :subunit_numericality
-                true
-              else
-                options[:subunit_numericality]
-              end
-
-              money_validation_options =
-              unless options.has_key? :numericality
-                true
-              else
-                options[:numericality]
-              end
-
               # This is a validation for the subunit
               validates subunit_name, {
                 :allow_nil => options[:allow_nil],
-                :numericality => subunit_validation_options
+                :numericality => options[:subunit_numericality] || true
```

Comment:
Agreed. Fetch is our friend here. 



- shane

On Mon, Mar 2, 2015 at 6:39 PM, Paul Nikitochkin <notifications@github.com>
wrote:

>>                # This is a validation for the subunit
>>                validates subunit_name, {
>>                  :allow_nil => options[:allow_nil],
>> -                :numericality => subunit_validation_options
>> +                :numericality => options[:subunit_numericality] || true
> what do you think about using `Hash#fetch` then `false` will be worked?
> By the way I think we does not have tests for `false` values, so you example passed but should not.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/RubyMoney/money-rails/pull/297/files#r25650303